src/Core_Command.php

```diff
@@ -158,6 +161,20 @@ public function download( $args, $assoc_args ) {
 			$download_url = str_replace( '.zip', '.tar.gz', $offer['download'] );
 		}
 
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) && 'en_US' !== $locale ) {
+			WP_CLI::error( 'The skip content build is only available for the en_US locale.' );
+		}
+
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) && isset( $assoc_args['version'] ) ) {
+			WP_CLI::error( 'The skip content build is only available for the latest version.' );
+		}
+
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) ) {
+			$offer = unserialize( self::_read( 'https://api.wordpress.org/core/version-check/1.6/' ) );
```

Comment:
Let's switch this to the v1.7 API

Reply:
Thanks for your feedback, I've just addressed this!